test/e2e/cypress/integration/single-backend/users.spec.js

```diff
@@ -314,10 +309,7 @@ describe('Users', function() {
       'contain',
       'Please add at least one profile'
     )
-    cy.get('[data-cy="UserUpdate-basicTab--dangerIcon"]').should(
-      'be',
-      'visible'
-    )
+    cy.get('[data-cy="UserUpdate-basicTab--dangerIcon"]').should('be.visible')
```

Comment:
To be consistent with other tests ?
```suggestion
    cy.get('[data-cy="UserUpdate-basicTab--dangerIcon"]').should('exist')
```

Reply:
No... Sometimes we want to check visibility and sometimes we want to check existence. This case is for visibility.